datasets/sick/sick.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{marelli-etal-2014-sick,
+    title = "A {SICK} cure for the evaluation of compositional distributional semantic models",
+    author = "Marelli, Marco  and
+      Menini, Stefano  and
+      Baroni, Marco  and
+      Bentivogli, Luisa  and
+      Bernardi, Raffaella  and
+      Zamparelli, Roberto",
+    booktitle = "Proceedings of the Ninth International Conference on Language Resources and Evaluation ({LREC}'14)",
+    month = may,
+    year = "2014",
+    address = "Reykjavik, Iceland",
+    publisher = "European Language Resources Association (ELRA)",
+    url = "http://www.lrec-conf.org/proceedings/lrec2014/pdf/363_Paper.pdf",
+    pages = "216--223",
+}
+"""
+
+_DESCRIPTION = """\
+Shared and internationally recognized benchmarks are fundamental for the development of any computational system.
+We aim to help the research community working on compositional distributional semantic models (CDSMs) by providing SICK (Sentences Involving Compositional Knowldedge), a large size English benchmark tailored for them.
+SICK consists of about 10,000 English sentence pairs that include many examples of the lexical, syntactic and semantic phenomena that CDSMs are expected to account for, but do not require dealing with other aspects of existing sentential data sets (idiomatic multiword expressions, named entities, telegraphic language) that are not within the scope of CDSMs.
+By means of crowdsourcing techniques, each pair was annotated for two crucial semantic tasks: relatedness in meaning (with a 5-point rating scale as gold score) and entailment relation between the two elements (with three possible gold labels: entailment, contradiction, and neutral).
+The SICK data set was used in SemEval-2014 Task 1, and it freely available for research purposes.
+"""
+
+_DOWNLOAD_URL = "https://zenodo.org/record/2787612/files/SICK.zip?download=1"
+
+
+class SICK(datasets.GeneratorBasedBuilder):
+    """The SICK (Sentences Involving Compositional Knowldedge) dataset."""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "sentence_A": datasets.Value("string"),
+                    "sentence_B": datasets.Value("string"),
+                    "entailment_label": datasets.features.ClassLabel(names=["ENTAILMENT", "NEUTRAL", "CONTRADICTION"]),
```

Comment:
```suggestion
                    "premise": datasets.Value("string"),
                    "hypothesis": datasets.Value("string"),
                    "label": datasets.features.ClassLabel(names=["entailment", "neutral", "contradiction"])
```
Would it be possible to have the columns defined this way ?
This is the standard we're using for NLI datasets

Reply:
The change for the class label column makes sense. I'm not sure about the change to `premise` and `hypothesis` because the dataset has other columns referring to the names "A" and "B" (e.g. also entailment from "B" to "A"), so `sentence_A` and `sentence_B` are not always strictly used as premise and hypothesis in every possible use case of SICK.

Reply:
Yes right ! Let's keep sentence_A and sentence_B then